staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1375,6 +1375,42 @@ type ScaleIOVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,10,opt,name=readOnly"`
 }
 
+// ScaleIOPersistentVolumeSource represents a persistent ScaleIO volume that can be defined
+// by a an admin via a storage class, for instance.
+type ScaleIOPersistentVolumeSource struct {
```

Comment:
this should be an exact copy of v1.ScaleIOVolumeSource with `SecretRef` changed to `*SecretReference`. That will pick up all the json/protobuf tags verbatim

Reply:
@liggitt ok, thanks.  I think I only changed `SecretRef`'s type in `ScaleIOPersistentVolumeSource`.  I will make sure they are the same in both `ScaleIOVolumeSource` as well.  Thanks.

Reply:
I re-read your comment.  Ok, I will ensure the types are the same in staging and v1.

Reply:
yeah, leave `ScaleIOVolumeSource` as is

Reply:
@liggitt 
I got confused because your comment above has `ScaleIOVolumeSource` using `SecretReference`.  Please confirm: 
* `XXXVolumeSource` -> `LocalObjectReference` (in pkg/v1 and staging/v1)
* `XXXPersistentVolumeSource` -> `SecretReference` (in pkg/v1 and staging/v1)

The above is how I see other types are done, but confirm anyway.

Reply:
Oops, fixed.